app/src/main/java/com/tvtracker/dao/IMediaItemDAO.kt

```diff
@@ -9,7 +9,9 @@ import retrofit2.http.Query
 interface IMediaItemDAO {
 
     @GET("/")
-    fun searchImdb(@Query("s") searchTxt: String, @Query("type") type: String, @Query("page") page: Int): Call<ImdbResponse>
+    fun searchImdb(@Query("s") searchTxt: String,
```

Comment:
Separated fun function signatures for readability
https://kotlinlang.org/docs/coding-conventions.html#functions